udmis/src/main/java/com/google/bos/udmi/monitoring/LogTailEntry.java

```diff
@@ -0,0 +1,28 @@
+package com.google.bos.udmi.monitoring;
+
+import com.google.cloud.logging.Severity;
+import java.time.Instant;
+
+public class LogTailEntry {
+
+  Instant timestamp;
+  String methodName;
+  String serviceName;
+  String resourceName;
+  int statusCode;
+  String statusMessage;
+  Severity severity;
+  int value = 1;
+
+  public boolean equals(LogTailEntry other) {
```

Comment:
There's a new thing in Java called a "record class" that makes some of this boiler-plate unnecessary -- see https://docs.oracle.com/en/java/javase/20/language/records.html -- don't care if you use it or not, but I was just reading about it so had to mention it!

Reply:
Nothing this as a TODO. e.g. If the equals() becomes more complex let's revisit.